make/release.mk

```diff
@@ -139,6 +141,7 @@ release-$1-zts: Makefile agent | releases/$$(RELEASE_OS)/agent/$$(RELEASE_ARCH)/
 
 endef
```

Comment:
I think this should be "endif"?

Reply:
When a define includes multiple lines, endef is used to show that the define is ending.
See here for more details: https://www.gnu.org/software/make/manual/html_node/Multi_002dLine.html

Reply:
Ah thanks. :)